### PR TITLE
[Behat] Regression script should wait for Enter when asking for confirmation

### PR DESCRIPTION
**Pull request by bogusez**
https://jira.ez.no/browse/EZP-31464
I have added a script which should wait for user confirmation to proceed regression script.

### CODE REVIEW

File: bin/.travis/trigger_ci.sh [hunk 1]
```diff
@@ -148,13 +148,12 @@ if [ "${HAS_PAGE_BUILDER_DEPENDENCY}" == 1 ] ; then
...
-echo ''
-
-if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
-  echo -e '\033[0;31mAborted.\033[0m'
-  exit 1
-fi
+read -p "Do you want to continue? [y/N]" -r
+if [[ ! $REPLY =~ ^[Yy] ]] ; then
+   echo -e '\033[0;31mAborted.\033[0m'
+   exit 0
```
Comment (mnocon) [inline bin/.travis/trigger_ci.sh:154]: We should exit with a non-zero return code, the script has been aborted (`exit 1` was ok)

File: bin/.travis/trigger_ci.sh [hunk 2]
```diff
@@ -148,13 +148,12 @@ if [ "${HAS_PAGE_BUILDER_DEPENDENCY}" == 1 ] ; then
...
+fi   
+echo 'Continue.'
```
Comment (mnocon) [inline bin/.travis/trigger_ci.sh:156]: Please remove this line

File: bin/.travis/trigger_ci.sh [hunk 3]
```diff
@@ -148,13 +148,12 @@ if [ "${HAS_PAGE_BUILDER_DEPENDENCY}" == 1 ] ; then
...
   echo -e "\033[1;33mRemember to talk with them that you are adding a commit to their branch and clearly mark the Pull Request as one containing temporary changes.\033[0m"
 fi
-read -p "Do you want to continue? " -n 1 -r
-echo ''
-
-if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
-  echo -e '\033[0;31mAborted.\033[0m'
-  exit 1
-fi
+read -p "Do you want to continue? [y/N]" -r
```
Comment (mnocon) [inline bin/.travis/trigger_ci.sh:151]:
```suggestion
read -p "Do you want to continue? [y/N]: " -r
```

This makes it easier to spot the entered value

File: bin/.travis/trigger_ci.sh (hunk 1 above)
Comment (mnocon) [inline bin/.travis/trigger_ci.sh:154]:
```suggestion
   exit 1
```

Comment (bogusez) [reply bin/.travis/trigger_ci.sh]: Done, now should be good

**Pull request merged**